### PR TITLE
DT-1559: delete group added to group details page

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**

### CODE REVIEW

File: views/groupDelete.njk [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <p class="govuk-body govuk-!-margin-bottom-5">All the user membership of this group will be removed</p>
+      <p class="govuk-body govuk-!-margin-bottom-9"><STRONG>This action cannot be undone</STRONG></p>
+      {% if hasMaintainAuthUsers %}
+
+        {{ govukButton({
+          text: "Delete group",
+          element: 'a',
+          href: maintainUrl + '/' + group + '/delete',
+          classes: "govuk-button--warning",
+          attributes: { "data-qa": "delete-group-button" }
```
Comment (petergphillips) [inline views/groupDelete.njk:42]: have a look at the other cancel button pages - I added some more spacing between the primary and secondary buttons

**Review by petergphillips — DISMISSED** [review]
LGTM

File: views/groupDelete.njk (hunk 1 above)
Comment (SimonMitchellMOJ) [reply views/groupDelete.njk:42]: will do thanks

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**